pom.xml

```diff
@@ -350,5 +350,29 @@
             <url>http://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
-</project>
+
+   <profiles>
+      <profile>
```

Comment:
The GitSCMTest profile allows me to limit tests to the GitSCM module.  It is intended to speed development and testing of changes to the GitSCM module.  This profile is entirely optional and can be deleted if it does not conform to plugin development practices.